service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java

```diff
@@ -375,7 +375,7 @@ void runSetWithSamException() throws Exception {
             "[ \"%s\", \"%s\", \"%s\" ]".formatted(recordId1, recordId2, recordId3));
 
     when(samClient.checkAuthAccessWithSam()).thenReturn(true);
-    doCallRealMethod().when(samService).hasComputePermission();
+    doCallRealMethod().when(samService).hasWritePermission();
```

Comment:
Did we check this was an expected change with identiteam?

Reply:
So this is just changing the test code to reflect what is actually being called. Turns out we have updated the code to check for write permission during POST requests instead of compute permission while back but the test code wasn't updated for some reason. Similar reasoning applies for the below change to hasWritePermission too.

In this particular test its calling POST /run_set endpoint which does check [hasWritePermission() here](https://github.com/DataBiosphere/cbas/blob/7a42398cfc13f1b98c493c88ef4cff7f25e2a8c3/service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java#L179-L181).